test/martha/martha_v3.test.js

```diff
@@ -251,6 +300,7 @@ test.serial('martha_v3 does not call Bond or return SA key when the host url is
 test.serial('martha_v3 returns null for fields missing in drs and bond response', async (t) => {
     // update the stub to return DRS response with missing fields only for this test
     sandbox.restore();
+    getJsonFromApiStub.onFirstCall().resolves(dataObjectServiceObject);
     getJsonFromApiStub = sandbox.stub(apiAdapter, getJsonFromApiMethodName);
     getJsonFromApiStub.onFirstCall().resolves(drsObjectWithMissingFields);
```

Comment:
Ah. I can see here. DOS goes with DOS. DRS always goes with DRS. So this should get renamed.